extractor/src/main/scala/com/digitalasset/extractor/json/JsonConverters.scala

```diff
@@ -3,92 +3,59 @@
 
 package com.digitalasset.extractor.json
 
-import com.digitalasset.daml.lf.data.{
-  FrontStack,
-  ImmArray,
-  Ref,
-  SortedLookupList,
-  Time,
-  Numeric => LfNumeric
-}
+import com.digitalasset.daml.lf.data.SortedLookupList
 import com.digitalasset.daml.lf.value.{Value => V}
+import com.digitalasset.daml.lf.value.json.ApiCodecCompressed
 import com.digitalasset.extractor.ledger.types.{Identifier, LedgerValue}
 import com.digitalasset.extractor.ledger.types.LedgerValue._
 import com.digitalasset.extractor.writers.postgresql.DataFormatState.MultiTableState
 import io.circe._
 import io.circe.generic.auto._
 import io.circe.generic.semiauto._
 import io.circe.syntax._
-import scalaz.@@
 
 object JsonConverters {
-  def toJsonString[A: Encoder](a: A): String = {
-    a.asJson.noSpaces
+  private[this] object LfValueSprayEnc
+      extends ApiCodecCompressed[String](
+        encodeDecimalAsString = true,
+        encodeInt64AsString = false
```

Comment:
shouldn't both be set to `true` by default? or is it set this way for backward compatibility? Do we really care about backward compatibility here?

Reply:
> shouldn't both be set to `true` by default?

No; the reasons true is good in other contexts are bad reasons here; in fact it would be better to make them both false here.

> is it set this way for backward compatibility?

Yes. 

> Do we really care about backward compatibility here?

Yes.

I don't know what queries have been written against extractor, so I'm minimizing the change while still accomplishing our goal of encoding unification.